test/prototype-test.js

```diff
@@ -0,0 +1,227 @@
+var QUnit = require("steal-qunit");
+
+var observe = require("../can-observe");
+var makePrototype = require("../src/-make-prototype");
+
+var canReflect = require("can-reflect");
+
+QUnit.module("can-observe prototype");
+
+QUnit.test("makePrototype basics", function(assert) {
+	function Obj() {}
+	Obj.prototype = makePrototype.observable(Object.create(Obj.prototype), {
+		observe: makePrototype.observable
+	});
+	var o = new Obj();
+
+	var sets = 0;
+	canReflect.onKeyValue(o, "fullName", function(value) {
+		if (++sets === 1) {
+			QUnit.equal(value, "Kevin");
+		} else {
+			QUnit.equal(value, "Connor");
+		}
+	});
+
+	o.fullName = "Kevin";
+	o.fullName = "Connor";
+});
+
+var classSupport = (function() {
+	try {
+		eval('"use strict"; class A{};');
+		return true;
+	} catch (e) {
+		return false;
+	}
+
+})();
+
+if(classSupport) {
```

Comment:
This won't work. If classes aren't supported this file will fail to parse entirely. However, if we are moving to Edge as the minimum browser support then everything supports classes now anyways.